crates/telemetry-subscribers/src/lib.rs

```diff
@@ -181,23 +280,28 @@ impl TelemetryConfig {
             self.crash_on_panic = true
         }
 
-        if env::var("MYSTEN_TRACING").is_ok() {
+        if env::var("TS_TRACING").is_ok() {
```

Comment:
Reading this I'm not quite sure what "TS" is. i think that these are probably worse names than them being prefixed with Mysten

Reply:
TS = Telemetry-subscribers (the name of this module).  I could just remove that prefix though, just like "TOKIO_CONSOLE" and "RUST_LOG"

Reply:
Ok I renamed a bunch of the env vars to be more readable